lib/bpk-component-text-link/README.md

```diff
@@ -0,0 +1,51 @@
+# bpk-component-text-link
+
+> Backpack React Native text-link component.
+
+## Installation
+
+Check the main [Readme](https://github.com/skyscanner/backpack-react-native#usage) for a complete installation guide.
+
+## Usage
+
+```js
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { spacingBase } from 'bpk-tokens/tokens/base.react.native';
+import BpkTextLink from 'backpack-react-native/bpk-component-text-link';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: spacingBase,
+  }
+});
+
+export default () => (
+  <View style={styles.container}>
+    <BpkText>
+        Check out the website <BpkTextLink accessibilityLabel="link to website" onPress={() => {}}>here</BpkTextLink>
+      </BpkText>
+  </View>
+);
+```
+
+## Props
+
+### BpkTextLink
+
+This component creates a clickable instance of the `BpkText` component and accepts the same props as [`BpkText`](/components/text/?platform=native), alongside the following:
```

Comment:
Touchscreens don't click*, this is backed up by the event handler being named `onPress`.

*Many tablets do have mouse support but they're still simulating a press.

```suggestion
This component creates a pressable instance of the `BpkText` component and accepts the same props as [`BpkText`](/components/text/?platform=native), alongside the following:
```